addon/utils/ce/paste-handler-helper-functions/cleanDocx.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  cleanListElements,
+  postCleanHtml,
+  preCleanHtml,
+  cleanEmptyElements,
+} from '@lblod/ember-rdfa-editor/utils/ce/paste-handler-helper-functions';
+
+const parser = new DOMParser();
+
+export function cleanDocx(html: string): string {
+  const document = parser.parseFromString(preCleanHtml(html), 'text/html');
+  const { body } = document;
+
+  cleanEmptyElements(body);
+  cleanListElements(body);
```

Comment:
why did you remove all the other cleaners here?

Reply:
yep, because I did lot of tests, and didn't see any impact to use other cleaners. We can return these without a problem, but just I don't think we will benefit anything doing that. If you want we can add any more tests, but on my testing experience everything seems to work without other cleaners as well.

Reply:
but like, did you test a document with footnotes? with links, multiple fonts, etc?